src/topology/bases.lean

```diff
@@ -809,6 +809,43 @@ end
 
 end sum
 
+section quotient
+
+variables {X : Type*} [topological_space X] {S : setoid X}
+omit t
+
+/-- The image of a topological basis under an open quotient map is a topological basis. -/
+lemma is_topological_basis.quotient {V : set (set X)}
+  (hV : is_topological_basis V) (h : is_open_map (quotient.mk : X → quotient S)) :
+  is_topological_basis (set.image (quotient.mk : X → quotient S) '' V) :=
+begin
```

Comment:
Could you first prove that for an arbitrary open quotient map and then specialize it to `quotient.mk`?

Reply:
If that's too annoying then don't worry about it, but if it's easy to do then I think we should do it

Reply:
Nice suggestion, thanks!